src/content/docs/kubernetes-pixie/k8s-otel/advance-config.mdx

```diff
@@ -0,0 +1,266 @@
+
+
+To send your Kubernetes telemetry data to multiple New Relic accounts simultaneously, inject your secondary ingest license key(s) into the OpenTelemetry Collector container and configure additional OTLP exporters.
+
+1. To inject your secondary license key(s):
+
+    * In the `env` section of your `values.yaml` file, add the following environment variable for each secondary ingest license key you want to use:
+
+      ```yaml
+      daemonset:
+        envs:
+          - name: MY_SECONDARY_LICENSE_KEY_VAR # Choose a descriptive environment variable name
+            valueFrom:
+              secretKeyRef:
+                name: <Your Secret Name> # Name of your Kubernetes Secret
+                key: <Your Secret Key>    # Key within the Secret that holds the license key
+      deployment:
+        envs:
+          - name: MY_SECONDARY_LICENSE_KEY_VAR
+            valueFrom:
+              secretKeyRef:
+                name: <Your Secret Name>
+                key: <Your Secret Key>
+      ```
+
+    * In the `envForm` section of your `values.yaml` file, add the following environment variable for each secondary license key you want to use:
+
+      ```yaml
+      daemonset:
+        envsFrom:
+          - secretRef:
+              name: <Your Secret Name>
+      deployment:
+        envsFrom:
+          - secretRef:
+              name: <Your Secret Name>
+      ```
+
+2. To add an `otlphttp` exporter in the `extraConfig` section for each additional account, referencing the injected environment variable:
+
+  ```yaml
+  daemonset:
+    configMap:
+      extraConfig:
+        exporters:
+          otlphttp/secondAccount: # Unique name for this exporter
+            endpoint: "{{include 'nrKubernetesOtel.endpoint'}}"
+            headers:
+              api-key: ${env:MY_SECONDARY_LICENSE_KEY_VAR} # Reference the env var
+  deployment:
+    configMap:
+      extraConfig:
+        exporters:
+          otlphttp/secondAccount: # Unique name for this exporter
+            endpoint: "{{include 'nrKubernetesOtel.endpoint'}}"
+            headers:
+              api-key: ${env:MY_SECONDARY_LICENSE_KEY_VAR} # Reference the env var
+  # Important: Add this exporter to the relevant pipelines below
+  pipelines:
+    metrics:
+      exporters:
+        - otlphttp/newrelic # Original exporter
+        - otlphttp/secondAccount # New exporter
+    traces:
+      exporters:
+        - otlphttp/newrelic
+        - otlphttp/secondAccount
+    logs:
+      exporters:
+        - otlphttp/newrelic
+        - otlphttp/secondAccount
+  ```
+
+  <Callout variant="tip">
+    You must also add the `otlphttp/secondAccount` exporter to the relevant `pipelines` (metrics, traces, logs) within your `extraConfig` for both the `daemonset` and `deployment` collectors to ensure data is actually sent through this new exporter.
+  </Callout>
+
+3. After updating your `values.yaml` file, apply the changes to your cluster:
```

Comment:
Do we need to mention that the changes in the values file need to be applied?
Isn't it assumed?
If you think so, then we should add this step to the other sections in the doc too